test/data/demo.layout.template

```diff
@@ -0,0 +1,136 @@
+{
+ "signatures": [
+  {
+   "keyid": "556caebdc0877eed53d419b60eddb1e57fa773e4e31d70698b588f3e9cc48b35",
+   "sig": "696c4f1e6d4de8b02cd94347f16536ffcd27c8b39b54d84884d205284e9594c6d88470c18f6c2b32259cbda9ebef3af933bc23031aeda25b681420df5b6da42c6fee132017e4d9156d3fef401c6a6d4a567848dc950a2d0874222b33ae4e66cd0aa3c0c9d33ed19e7fe54fe5b78404ed2a1ac2bb648dfe5c19ffe6837b4bb1ffb4b8eacd7ebb80caef1ad96b723636fdb84bccd076f49d4fefce2c09dbc1b297c1d38c08a48ed1c5231c711722d744f33a7bc06b5291270f8e3e8cf3c2ee28aba53a6cd62e1171ba8c77b43d6fd85dada544b606e52c7bab840e0ff2793f2fdc0fa86c018c8e37dc0cd0ea23a4678c1a7689b58f710bbcd81f3a4f1b04b130978aaa77727f8afdd385b5a6dfc36e6ea638bf74c68acdfd00e72dd6614b20d57ab99e36ea171657eb2694cbecb4665eb8006fbe8e03d9c5e090d0c19c6a93db484a59c0ba9c2bb2bcb0e5c69c4732de3365859f52ac75c49840601155c4a1bebadf2759db9f308bdb0b5d628177ee7cea6953a3f31049b8f52b3c5007c5fe1cbb"
+  }
+ ],
+ "signed": {
+  "_type": "layout",
+  "expires": "2018-11-18T16:06:36Z",
```

Comment:
Happy testing in three weeks' time :-)

Reply:
On a more serious note, can't / shoudn't the keys, tarballs etc. be created on-the-fly for testing?

Reply:
Good eyes! But did you not see the note in commit message for f0c23f1e37a92ae4538217bbc02c63c425614d25?! (just kidding) You're right, these things (layout expiration date and signature) should be added on the fly, as we do in the reference implementation tests. The only reason why I hardcoded the date and pre-signed the layout for now is because I haven't implemented signing yet (not required for in-toto verification). I set myself a three weeks time window to fix this. :)

I also meant to add a comment, but there's no commenting in JSON. I could add a todo note in `test_verifylib.go`.

Reply:
How could I miss that :-)

Let's add a todo note, or file an issue.

Reply:
I added one in the verifylib test da9a05e